### ISSUE TITLE
Dive_demo does not display any grid while Overview_demo works fine

### ISSUE THREAD

**Issue by Imane0** [PAIR-code/facets#142]
I executed the provided notebooks [facet_overview](https://github.com/PAIR-code/facets/blob/master/facets_overview/Overview_demo.ipynb) and [facets_dive](https://github.com/PAIR-code/facets/blob/master/facets_dive/Dive_demo.ipynb) on my own dataset. 

`facets_overview` works as expected (interactive output, all statistics, all options for `Chart` and `Sort by`, etc...). 

However `facets_dive` outputs no grid at all. See image below.

![facets_dive_error](https://user-images.githubusercontent.com/10118778/42883010-a4cf7958-8a9a-11e8-8625-73e1761ab6a8.png)

With `Ctrl-Shift-i` in the browser where my notebook is open, I can read the following errors (cf right bottom corner in the above image, written in red):
```

Uncaught TypeError: Cannot read property 'findSprites' of undefined
    at a.clicked (facets-jupyter.html:7063)
    at SVGSVGElement.<anonymous> (facets-jupyter.html:169)
facets-jupyter.html:7066 Uncaught TypeError: Cannot read property 'width' of undefined
    at a.fitToViewport (facets-jupyter.html:7066)
    at HTMLElement.fitToViewport (facets-jupyter.html:7120)
    at HTMLElement.a.fitButton.onclick (facets-jupyter.html:8155)
2facets-jupyter.html:7063 Uncaught TypeError: Cannot read property 'findSprites' of undefined
    at a.clicked (facets-jupyter.html:7063)
    at SVGSVGElement.<anonymous> (facets-jupyter.html:169)
```

How can I solve this problem and make Dive output a grid ?

**Comment by Imane0** [comment]
I have the same output even when I open the notebook with `jupyter notebook --NotebookApp.iopub_data_rate_limit=10000000 notebook.ipynb`
I further noticed when scrolling up the Console that I have the following errors:
```
Failed to load resource: the server responded with a status of 404 (Not Found)
Failed to load extension: Array(1) Error: Script error for "nbextensions/facets-dist/"
http://requirejs.org/docs/errors.html#scripterror
    at makeError (require.js:165)
    at HTMLScriptElement.onScriptError (require.js:1732)

```

**Comment by jameswex** [comment]
can you try changing you notebook's HTML_TEMPLATE to pull facets dive from github as opposed to your locally installed facets dive dist HTML?
Use this link tag in your html template and let me know if that changes anything.  Thanks.

<link rel="import" href="https://raw.githubusercontent.com/PAIR-code/facets/master/facets-dist/facets-jupyter.html">

**Comment by livington** [comment]
I'd had the same error, I was trying to use facets dive from local machine and from github repo too.

![2019-01-29 18-56-19](https://user-images.githubusercontent.com/29087895/51921009-9417e900-23f7-11e9-9807-f632940ed55a.png)

![2019-01-29 18-54-50](https://user-images.githubusercontent.com/29087895/51920937-6e8adf80-23f7-11e9-9c45-1da4122072fb.png)

**Comment by MarineN1** [comment]
I have the same issue, how to fix this problem ? 
Thanks

**Comment by jimbojw** [comment]
I'm not sure, but it seems like there could be different causes that have the same symptoms. @Imane0 's original post said that the Demo page worked fine.

@livington and @MarineN1 , does the demo page work for you? Or does that also exhibit the problem?  https://pair-code.github.io/facets/

**Comment by MarineN1** [comment]
Yes, in fact, contrary to @Imane0, the demo page doesn't work for me, I have the same display problem as @livington.

**Comment by jimbojw** [comment]
@MarineN1 Ok thanks, can you try this THREE.js demo page too? https://threejs.org/examples/#webgl_animation_cloth

**Comment by MarineN1** [comment]
The THREE.js demo page works, I see the animations. 
Furthermore as @Imane0 facets_overview works as expected with interactive output

**Comment by MarineN1** [comment]
I have just the problem with facets_dive

**Comment by MarineN1** [comment]
No solution to fix the problem with facets_dive?
Thanks
